index.html

```diff
@@ -42,6 +41,133 @@ <h1><span class="color-blue"> Hey there. <br> I'm Ale. </span><br><span id="titl
             </section>
             <a class="arrow" href="#"><img src="./images/Disabled.svg" alt=""></a>
         </section>
+        <section class="works">
+            <div class="title"> 
+                <h1 class="color-blue">My Recent Works</h1>
+            </div>
```

Comment:
- [ ] Following the best practices, you are meant to have only one `h1` tag on your page and that should be the main title, this is not the main title, and therefore ti should be in `2h or h3` tag, I'll suggest you update this accordingly. :+1: :smile: